### DISCUSSION TITLE
Explanation of the parameters in the config.yaml file

### DISCUSSION THREAD

**Discussion by jackieway** [metagenome-atlas/atlas#709] [General]
Hello, I'm very new to learning bioinformatics and for a project, I've been tasked to analyze sequencing data from 12 environmental samples. These samples were enriched for various bacterial species and methanogens, but the identities of these species are mostly unknown (we're trying to get a better idea of these samples' genomes). I was wondering how I should be editing the config.yaml file that is produced, and how I can adjust the parameters in a way that would make sense for analyzing environmental bacteria and archaea samples that went through shotgun sequencing (if the parameters need to be adjusted at all, or if the default parameters should be used and that by itself is fine).

I was also wondering if I could get a general explanation for all of the parameters in the generated config.yaml file and what they mean (for example, "preprocess_minimum_base_frequency" in the quality control section, "megahit_merge_level" in the assembly section, "genome_filter_criteria" in the binning section, etc). Or maybe any literature or other resources that I could read to get a better idea of what these parameters mean. I've read the "configure atlas" portion of the metagenome-atlas documentation, but don't think I have the background information to fully understand it. Thanks!

**Comment by SilasK** [comment]
As I understand your project: you don't have a host to remove and you have metagenome samples.
Then the default parameters are all you need. Atlas is routinely used to assemble methanogens. 

Notice there is a tutorial for atlas. And maybe a caveat is that the `atlas run genome`  command gives you one representative genome per species. You might also want to look at all the bins produced by the intermediary `atlas run binning`.

You can study all the parameters, but I think the config arguments you don't understand with the comments in the template are not so important. 
- [ ] I should simplify the template config.

There is a Configure section in the docs:
https://metagenome-atlas.readthedocs.io/en/latest/usage/configuration.html#

Which should display a detailed configuration section, but doesn't correctly. You can find it here:
https://github.com/metagenome-atlas/atlas/tree/master/docs/advanced
- [ ] I will fix this doc issue.

**Reply by jackieway** [reply]
Thanks for the response! I'll read over the detailed configuration section on github as well
